config/config/sc-config.yaml

```
@@ -297,6 +297,14 @@ elasticsearch:
   #   definition:
   #    users:
   #      - "Developer Name"
+  #  - mapping_name: kibana_user
+  #    definition:
+  #      backend_roles:
+  #        - "kibana_dev"
```

I'd also map a dev to `kubernetes_log_reader` as to be able to view logs in kibana :)
```suggestion
  #  - mapping_name: kibana_user
  #    definition:
  #      backend_roles:
  #        - "kibana_dev"
  #  - mapping_name: kubernetes_log_reader
  #    definition:
  #      backend_roles:
  #        - "kibana_dev"
```

---

Actually both users when they log in get that role of kubernetes_log_reader (seethe issue here with screenshots:https://github.com/elastisys/compliantkubernetes-apps/issues/98

But is that the behaviour we actually want? (everyone getitng that role)

---

> Actually both users when they log in get that role of kubernetes_log_reader (seethe issue here with screenshots:#98
> 
> But is that the behaviour we actually want? (everyone getitng that role)

So this was a dirty config on my side, resolved